src/lib/descriptor/mod.rs

```diff
@@ -14,69 +14,175 @@ mod cargo_alias;
 pub(crate) mod descriptor_deserializer;
 mod makefiles;
 
-use crate::io;
-use crate::plugin::descriptor::merge_plugins_config;
-use crate::scriptengine;
-use crate::types::{
-    Config, ConfigSection, EnvFile, EnvFileInfo, EnvValue, Extend, ExternalConfig, ModifyConfig,
-    Task,
-};
-use crate::version;
-use envmnt;
+use std::collections::HashSet;
```

Comment:
what is this custom 'use' order? i put everything and let rustfmt (cargo make format) order it automatically.

Reply:
Oh yes, do you use a custom rustfmt config by any chance? There was no `rustfmt.toml` in the project directory, so it must've picked up my default configuration.

Reply:
I don't use any custom formatting. only default one on purpose.

Reply:
would it be okay to just create an empty `rustfmt.toml` in the root directory? That way - even if people (like me) - have a custom configuration. It won't take precedence over the default formatting rules.